apps/explorer/src/ui/Button.tsx

```diff
@@ -29,15 +30,27 @@ export interface ButtonProps
     extends VariantProps<typeof buttonStyles>,
         ButtonOrLinkProps {
     loading?: boolean;
+    before?: ReactNode;
+    after?: ReactNode;
 }
 
 export function Button({
     variant,
     size,
     loading,
     children,
+    before,
+    after,
     ...props
 }: ButtonProps) {
+    const childrenContent = (
```

Comment:
Just inline this instead of making it a variable

Reply:
It's used in 2 different places. If I only pass `children` in on the `text-transparent` div, it will shift around. I am doing this so I dont have to copy/paste the same block in 2 places


![switch](https://user-images.githubusercontent.com/127577476/225181668-ced4ac1f-4a17-4e4e-949c-ad31ce241868.gif)

Reply:
Ah I missed that. Yeah I think there's opportunity to unify this but this is fine for now

Reply:
@Jordan-Mysten I cleaned up the `Button` file, it should be more streamlined now